packages/components/src/color-palette/types.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import type { MouseEventHandler, ReactElement } from 'react';
+
+type OnColorChange = ( newColor?: string ) => void;
+
+export type Color = {
+	name: string;
+	color: string;
+};
+
+export type MultipleColors = {
+	name: string;
+	colors: Color[];
+}[];
+
+type PaletteProps = {
+	className?: string;
+	clearColor: () => void;
+	onChange: OnColorChange;
+	value?: string;
+	actions: ( ReactElement | null ) | ( false | ReactElement );
```

Comment:
We should be able to just use `ReactNode` for this:

```suggestion
	actions?: ReactNode;
```

Reply:
That's much better, thanks!

Committed in https://github.com/WordPress/gutenberg/pull/44632/commits/c9f5d8a2f85a36641d557aae049ac2dbcb0ab6c0

Reply:
Apologies in case it wasn't clear, but my suggestion was also to make `actions` non-required (via the `?` syntax) — could we do that?

Reply:
Sorry, your suggestion was pretty clear! I'll commit that now.

Reply:
Just committed in https://github.com/WordPress/gutenberg/pull/44632/commits/a07c43fbc934b2c75504c29fc81fcf1fabcd2664